scripts/python/migrate_transaction_status_date.py

```
@@ -0,0 +1,217 @@
+#!/usr/bin/env python3
+import os
+import sys
+import boto3
+import logging
+from datetime import datetime
+from decimal import Decimal
+from typing import Dict, Any, Optional
```

Ruff (0.11.9)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)